app/src/main/java/com/example/musicapp/service/InstrumentService.kt

```diff
@@ -11,31 +11,26 @@ import android.util.Log
 
 class InstrumentService {
 
-    fun fetchInstruments() : MutableLiveData<ArrayList<Instrument>>{
+    internal fun fetchInstruments(): MutableLiveData<ArrayList<Instrument>> {
```

Comment:
Code Review Suggestion 2:
I added the internal visibility modifier here so that this function isn't available outside of the module. This is an easy mistake to make but having these functions set to the default public modifier could cause issues as they are available anywhere. The link to information about these visibility modifiers is https://kotlinlang.org/docs/reference/visibility-modifiers.html